src/test/ui/const-generics/condition-in-trait-const-arg.rs

```diff
@@ -1,7 +1,10 @@
+// Checks that `impl Trait<{anon_const}>` evaluates successfully.
```

Comment:
```suggestion
// Checks that `impl Trait<{anon_const}> for Type` evaluates successfully.
```

`impl Trait` has a slightly different meaning :sweat_smile: